common/Mailer.js

```diff
@@ -1,56 +1,39 @@
-"use strict";
-
-var validator = require('validator');
-var nodemailer = require('nodemailer');
-var mandrillTransport = require('nodemailer-mandrill-transport');
-var emailTemplates = require('email-templates');
-var templatesDir = require('path').resolve(__dirname, '..',  "email-templates");
-
-var credentialMandrill = require('common/makeCredential')('mandrill');
-var log = require('common/log');
-
-function Mailer(opts){
-	// 必須項目のチェック
-	// if(!opts.subject){ throw new Error("Mailer must have subject."); }
-	// if(!opts.from){ throw new Error("Mailer must have from mail-address."); }
-	// if(!opts.to){ throw new Error("Mailer must have to mail-address."); }
-	// if(!opts.text && !opts.html){
-	// 	throw new Error("Mailer must pass text or html.");
-	// }
-
-	// バリデーション
-	// if(!validator.isEmail(opts.from) || !validator.isEmail(opts.to)){
-	// 	throw new Error("to and from should exactly mail-address.");
-	// }
-
-	this.UserCollections = require('models/User');
-	this.BookListCollections = require('models/Book');
-
-	this.subject = opts.subject;
-	this.from = opts.from;
-	this.to = opts.to;
-	this.text = opts.text || "";
-	this.html = opts.html || "";
+const nodemailer = require('nodemailer');
+const mandrillTransport = require('nodemailer-mandrill-transport');
+const emailTemplates = require('email-templates');
+const templatesDir = require('path').resolve(__dirname, '..', 'email-templates');
+
+const credentialMandrill = require('common/makeCredential')('mandrill');
+
+function Mailer(opts) {
+  this.UserCollections = require('models/User');
+  this.BookListCollections = require('models/Book');
+
+  this.subject = opts.subject;
+  this.from = opts.from;
+  this.to = opts.to;
+  this.text = opts.text || '';
+  this.html = opts.html || '';
 }
 
-Mailer.prototype.send = function(done){
-  var transporter = nodemailer.createTransport(mandrillTransport({
+Mailer.prototype.send = function send(done) {
+  const transporter = nodemailer.createTransport(mandrillTransport({
     auth: {
-      apiKey: credentialMandrill
-    }
+      apiKey: credentialMandrill,
+    },
   }));
 
-  var mailOptions = {
-    from: "kindelize.it <" + this.from + ">",
+  const mailOptions = {
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).